### DISCUSSION TITLE
Any updates about Alpine.js integration?

### DISCUSSION THREAD

**Discussion by Blinks44** [tailwindlabs/headlessui#766] [General]
Hi guys! Just want to say, that you are awesome!

Any updates about headlessui + Alpine.js?

**Comment by argirisp** [comment]
Second that, I think that Alpine lives in the _goldilocks zone_ among "write your own js", "add the React/Vue complexity" (even for a single one page landing page) and a "tailwind.js" which would make assumptions (like bootstrap.js).

**Comment by mattgreenfield** [comment]
It looks like the answer is 'not any time soon' -  https://github.com/tailwindlabs/headlessui/discussions/344#discussioncomment-961311

**Comment by divine** [comment]
Hello,

No point to have headless UI for Alpine as Caleb is working on it https://alpinejs.dev/patterns

> Jason and I are building out a bunch of common, UI components that most people need in their projects. We'll of course cover the essentials like modals, dropdowns, and tabs, but we'll also tackle deeper, more specific ones as we research more about what people need. (more on this later)
> If you've ever tried building any of these components yourself, by hand, you know how insanely tough it is to do properly. After taking into account keyboard and screen-reader accessibility, responsiveness, and cross browser compatibility, what starts as a simple task becomes a massive undertaking.
> We're doing all this work so you don't have to. Just copy and paste our code into your app and get going.
> (We'll provide minimal styling for the basic structure, but leave things like colors, font choices, and border radiuses up to you. This way you don't have to wrestle with our markup to make these components fit within your app.)

Thanks!
